tests/foreman/cli/test_user.py

```diff
@@ -587,3 +587,89 @@ def test_negative_personal_access_token_invalid_date(self, target_sat):
                 action='create',
                 options={'name': token_name, 'user-id': user['id'], 'expires-at': datetime_expire},
             )
+
+
+@pytest.mark.no_containers
+@pytest.mark.parametrize('auth_type', ['admin', 'non-admin'])
+@pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
+def test_positive_invalidating_users_tokens(
+    module_target_sat, rhel_contenthost, module_activation_key, module_org, auth_type
+):
+    """Verify invalidating single and multiple users tokens.
+
+    :id: 5db602d4-9c57-4b70-8d46-5323044824e0
+
+    :steps:
+        1. Create an admin user and a non-admin user with "edit_users" and "register_hosts" permission.
+        2. Generate a token with admin user and register a host with it, it should be successful.
+        3. Invalidate the token and try to use the generated token again to register the host, it should fail.
+        4. Invalidate tokens for multiple users with "invalidate-multiple" command, it should invalidate all the tokens for provided users.
+        5. Repeat Steps 2,3 and 4 with non-admin user and it should work the same way.
+
+    :expectedresults: Tokens invalidated cannot be used for registration of hosts.
+
+    :CaseImportance: Critical
+
+    :Verifies: SAT-30385
+    """
+    admin_login = gen_string('alpha')
+    non_admin_login = gen_string('alpha')
+    password = gen_string('alpha')
+    login = admin_login
+    admin_user = module_target_sat.cli_factory.user(
+        {
+            'login': admin_login,
```

Comment:
Why not using settings.server.admin_username here instead?

Reply:
Yes, but any specific reasons to not use it?

Reply:
We are going to invalidate the tokens of the user we are creating and I don't want to modify any behavior of default admin user.